### PR TITLE
Pmeshram/sap/updates - Workload Specific Recommendations - SAP 

### PR DESCRIPTION
**Pull request by PmeshramPM**
# Overview/Summary

I have updated the APRL workload specific recommendations for SAP and reduced the content. 

## This PR fixes/adds/changes/removes

1. I have updated the APRL workload specific recommendations for SAP and reduced the content. 

### Breaking Changes

None

## As part of this Pull Request I have

- [ ] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [x] Ensured PR tests are passing
- [ ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Review by rodrigosantosms — CHANGES_REQUESTED** [review]
This page should not be modified as it is populated automatically by Hugo. Please review the contribution guide and create a Page for SAP, and then submit the content to the SAP Page, not the Services List page. thank you

File: docs/content/services/specialized-workloads/sap-on-azure/_index.md
Comment (oZakari) [inline docs/content/services/specialized-workloads/sap-on-azure/_index.md:1]: Could you adjust the format of this page to align to the baseline template. The contribution guide has [steps ](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#creating-a-services-recommendation-page)for how to generate it for your service.

File: docs/content/services/specialized-workloads/_index.md
Comment (oZakari) [inline docs/content/services/specialized-workloads/_index.md:1]: Please revert this change as this is necessary for the Specialized Workloads category to be visible within APRL.

**Review by oZakari — CHANGES_REQUESTED** [review]
Hi @PmeshramPM, thanks for the work on this. Once you make the changes below, please validate locally to make sure everything is formatted and showing up as you expecct. You can follow these [steps](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#softwareapplications) to install hugo locally. 

Please feel free to ping me if you have any questions or need help with anything!

**Comment by PmeshramPM** [comment]
@oZakari @rodrigosantosms - I am cancelling this PR. We have added a list of recommendations in another PR(https://github.com/Azure/Azure-Proactive-Resiliency-Library/pull/283)
